Report/Template.Rmd

```diff
@@ -243,17 +251,267 @@ datasets = read.and.clean(dataset = "easySHARE_rel6_0_0.rda", wav = 99)
+  
+  if(class(testOutput) == "try-error"){
+    msg = paste0("Wald Test failed for Model Element ", i)
+    warning(msg)
+    wald.log[[i]] = "Error"
+  } else{
+    wald.log[[i]] = testOutput
+  }
+  rm(SummaryElement) # clean up
+}
+
+wald.bound = as.data.frame(wald.log)
+colnames(wald.bound) = names(allModels)
+
+```
+
+
+Furthermore, the application of the Wald Test can easily be substitued. We use a different Wald Test to validate our results. 
+
+```{r, eval = FALSE}
+    testOutput = try(wald.test(b = coef(ModelElement), 
+                               Sigma = vcov(ModelElement), Terms = health)$result)
+```
+
+In line with the proposed concatenated `lapply` structure, the probability of employment can be predicted for each subset. Marginal effects are calculated using `probitmfx`, which firstly recalculates a probit model. 
+
+```{r, eval = FALSE, warning = FALSE}
+# Calculate marginal effects and standard errors
+
+# Same structure as before, but must calculate model again. 
+mfx.Models = lapply(df.splits, function(z){
+    
+    z   = z[-z$age50] # Multicollinearity
+    res = probitmfx(z$labor_participationTRUE ~.,atmean = TRUE,  data = z)
+    
+    return(res)
+})
+
+```
+
+
+## Empirical Results
+
+By means of the presented model, it is evaluated how labor force participation behavior of individuals aged 50-64 is influenced by different health indicators across 11 European countries.
+The underlying data enables us to mitigate justification bias and measurement error that occur when assessing health subjectively [@kalwij2005labour]. 
+
+@kalwij2005labour summarize the results as follows:   
+
+- The influence of health (as captured by the indicators) on labor force participation differs among individuals and across countries
+- Health improvements are associated with higher labor force participation rates
+- Decreasing health explains decreasing labor force participation of the elderly in Europe
+
+These main results are validated by the analysis. As shown on the example below, which is a summary of the probability of Austrian women participating in the labor market as explained by the probit model, age and the prevalence of chronic diseases negatively related to employment. Analysis of the marginal effects show, that the negative influence of age on the dependent variable is growing with increasing age. Conversely, high education positively predicts labor force participation for the specified group. Having children does not significantly influence the independent variable. These results are stable across different groups. 
+
+Comparing the marginal effects between our results and the paper by @kalwij2005labour shows similar results. Usually, significant coefficient estimates as well as their significance level are close to each other. However, as the data sets differ, there are deviations in the majority of models. This rarely affects the sign of the coefficients. 
```

Comment:
Didnt you just compare marginal effects?

Reply:
Yes sorry. Don't know how I couldnt see that...